arrow-libs/optics/arrow-optics/src/commonMain/kotlin/arrow/optics/Prism.kt

```diff
@@ -172,6 +174,26 @@ public interface PPrism<S, T, A, B> : POptional<S, T, A, B>, PSetter<S, T, A, B>
         getOrModify = { option -> option.fold({ Either.Right(Unit) }, { Either.Left(option) }) },
         reverseGet = { _ -> None }
       )
+
+    /**
+     * [Prism] to focus into an [arrow.core.Either.Left]
+     */
+    @JvmStatic
+    public fun <L, R> pLeft(): Prism<Either<L, R>, L> =
```

Comment:
It's prefixed `p` but implement non-polymorphic Optic. The pattern is prefix with `p` for `PPrism` and expose non-prefixed variant for regular `Prism`.

Reply:
Oh, I misunderstoof the convention, I thought `p` stood for "prism". The problem here is that `right` and `left` are already taken by other operations in `PPrism`; any ideas of better naming?

Reply:
Fixed in the latest commit